package friends_or_foes;

message HexCellProto {
  required int32 horiz_coord = 1;
  required int32 diag_coord = 2;
}

message HexPopulationEvent {
  enum Type {
    stop = 0;
    kill = 1;
    replicate = 2;
  }
  required Type type = 1;
  optional HexCellProto cell0 = 2;
  optional HexCellProto cell1 = 3;
}
